### ISSUE TITLE
Dedicated error page if a room doesn't exist

### ISSUE THREAD

**Issue by schiessle** [nextcloud/spreed#5070]
Following situation which I had already multiple times:

I invite someone to a Nextcloud Talk meeting. For some reason they don't copy the complete URL (e.g. the mail client added a line break within the URL, the calendar shows only part of the URL in the tooltip and they copy&paste it from there,...)

In this case the user gets redirected to the login page and is completely irritated. If they don't know Nextcloud it looks like they need a username and password to join the meeting.

Therefore I would suggest to have a dedicated error page, something along the line "The room you are trying to access doesn't exist. Please check the URL". So that the user knows what (most likely) went wrong and can act acordingly

cc @nextcloud/designers

**Comment by nickvergessen** [comment]
So previously the take was "We should redirect to login, because it's irritating that you see an error, when you actually just got logged out by accident."

**Comment by schiessle** [comment]
I see the argument for internal users. On the other hand for external users that's super confusing.

**Comment by schiessle** [comment]
Maybe a more verbose message does the trick (explaining what is going on is always good):

"The room you are trying to access doesn't exist or is a internal room. 
Please check the URL and try again.

If you have an account, login and try again [link to login]"

(last sentence only if the user is not logged in).

I think we should try to come up with an error handling which works for both.

**Comment by jancborchardt** [comment]
Yup, @schiessle’s proposal with the more verbose message sounds very good! Slight wording improvements:

> **Conversation not found**
> This conversation does not exist or is not shared. Please check the link.
> If you have an account, log in:
> (  Log in  )

("Log in" is a button)
